docs/src/main/sphinx/release/release-379.md

```diff
@@ -0,0 +1,97 @@
+# Release 379 (27 Apr 2022)
+
+## General
+
+* Add {doc}`/connector/mariadb`. ({issue}`10046`)
+* Improve performance of queries that contain `JOIN` and `UNION` clauses. ({issue}`11935`)
+
+## Security
+
+## Web UI
+
+## JDBC driver
+
+## Server RPM
+
+## Docker image
+
+## CLI
+
+## Accumulo connector
+
+## Atop connector
+
+## BigQuery connector
+
+## Blackhole connector
+
+## Cassandra connector
+
+## ClickHouse connector
+
+## Delta Lake connector
+
+* Add support for Google Cloud Storage. ({issue}`12144`)
+* Fix failure when reading from `information_schema.columns` when non-Delta
+  tables are present in the metastore. ({issue}`12122`)
+
+## Druid connector
+
+## Elasticsearch connector
+
+## Google Sheets connector
+
+## Hive connector
+
+* Allow Glue Iceberg/Delta compatibility even when Glue storage properties exist. ({issue}`12130`)
```

Comment:
@dain @erichwang ..  I wasnt sure if this applies to Hive connector only from the PR or also others .. please clarify

Reply:
Ping @dain @erichwang

Reply:
@mosabua, this actually should be updated based on this fix we are trying to put in:
https://github.com/trinodb/trino/pull/12164

```
* Improve Hive connector Glue Iceberg compatibility when other Glue storage properties exist.
```

Reply:
Done in a call with @erichwang